pkg/crypto/crypto_test.go

```diff
@@ -37,7 +38,10 @@ func TestNewAddress(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	a := crypto.NewOverlayAddress(k.PublicKey, 1)
+	a, err := crypto.NewOverlayAddress(k.PublicKey, 1)
```

Comment:
how is it that here `NewOverlayAddress` is called with `1` and in the test above it is `3`?

Reply:
no idea, I just left is as it was

Reply:
k

Reply:
@acud  That is a networkID argument. If you think the tests in bzz/address_test.go by test above, I think I just used number 3 randomly :)